newton_usd_schemas/generatedSchema.usda

```diff
@@ -2,3 +2,23 @@
 (
 )
 
+class NewtonSceneAPI "NewtonSceneAPI" (
+    doc = "NewtonSceneAPI applies on top of a PhysicsScene Prim, providing attributes to control a Newton Solver."
+)
+{
+    uniform double newton:maxSolverIterations = 100 (
+        doc = "Maximum number of iterations of the physics solver"
+    )
+
+    uniform double newton:timeStep = 0.005 (
+        doc = "Simulation time step in seconds"
+    )
+
+    bool newton:enableGravity = true (
+        doc = """Enable/disable gravity in the simulation.
+
+        This is intentionally separated from the gravity direction and magnitude
+        to allow temporary disabling of gravity without modifying & caching values.
+        """
```

Comment:
I need to investigate whether this multi-line doc string renders nicely in various DCCs.

Reply:
Looks good in Houdini too

<img width="1782" height="628" alt="image" src="https://github.com/user-attachments/assets/580416a8-d773-4021-855b-af49fc87f5c2" />